GenderPayGap.WebUI/Areas/Account/Views/ManageAccount/ManageAccount.cshtml

```diff
@@ -2,8 +2,11 @@
 
 @await Html.PartialAsync("AccountNavigation")
 
+@{ await Html.RenderPartialAsync("../../../../Views/Components/StatusMessages/StatusMessages"); }

```

Comment:
@jamesgriff 
This shows like this:
![image](https://user-images.githubusercontent.com/15341769/94447316-88b72b00-01a1-11eb-99fb-6cbc40a58726.png)

To me, the spacing looks a bit odd here, but I wasn't sure how to handle it - do you have any suggestions?